package.json

```diff
@@ -19,7 +19,6 @@
 		"build-client": "gulp",
 		"build-concurrently": "yarn install-if-deps-outdated && yarn clean && yarn concurrently 'yarn build-client' 'yarn build-php' 'yarn build-extensions' 'yarn build-search' 'yarn build-packages'",
 		"build-extensions": "webpack --config ./webpack.config.extensions.js",
-		"build-jetpack": "cd projects/plugins/jetpack && yarn build",
```

Comment:
Not-blocking: What's the preferred way to build Jetpack if you're in the repo root with this removed?

Reply:
FWIW, I added it back in #18212 along with cleaning up all the non-working things in this file.